src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst

```diff
@@ -2351,3 +2351,70 @@ Sorting
 Testing
 -------
 All queries above are (or should be) defined as integration tests under the namespace `Pim\Bundle\CatalogBundle\tests\integration\PQB`.
+
+
+Id
```

Comment:
Can you explain a bit more whats this id is about ? (maybe what does it correspond to in Mysql)